shpc/tests/test_version_naming.py

```diff
@@ -69,7 +69,7 @@ def test_disabled_version_naming_install(
     client = init_client(str(tmp_path), module_sys, container_tech)
 
     client.settings.set("version_naming", False)
-    assert client.settings.set("version_naming", False) is False
+    assert client.settings.set("version_naming") == False
```

Comment:
You need to first `set` it, and then use `get` to check the setting is what you set it to.